src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -414,12 +442,22 @@ export function fetchPastAppointments(startDate, endDate, selectedIndex) {
       });
 
       try {
-        const facilityData = await getAdditionalFacilityInfo(
-          getState().appointments.past,
-          featureFacilitiesServiceV2,
-        );
+        let facilityData;
+        if (
+          featureVAOSServiceVAAppointments ||
+          featureVAOSServiceCCAppointments
```

Comment:
Same comment as above, I think you can skip checking `featureVAOSServiceCCAppointments`.